ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -35,6 +38,25 @@ class DataExplorer extends Component {
     return queryConfigs[0]
   }
 
+  componentDidMount() {
+    const {query} = queryString.parse(window.location.search)
+    if (query && query.length) {
```

Comment:
Let's limit the amount of times we use the window object.  Let's use `router.location.search` here instead.

Reply:
oh right! my oversight 😸